views/default/resources/widgets/add_panel.php

```diff
@@ -18,6 +20,9 @@
 
 $context = (string) elgg_extract('context', $vars, get_input('context'));
 $owner_guid = (int) elgg_extract('owner_guid', $vars, (int) get_input('owner_guid'));
+$new_widget_column = elgg_extract('new_widget_column', $vars, get_input('new_widget_column'));
+$new_widget_position = elgg_extract('new_widget_position', $vars, get_input('new_widget_position'));
```

Comment:
default??

Reply:
nope... if i default it will be added to the urls.. this way it stays null